program/tests/test_create_mint.rs

```diff
@@ -0,0 +1,450 @@
+use solana_program::program_error::ProgramError;
+use solana_program::program_pack::Pack;
+use solana_program::rent::Rent;
+use solana_program::system_program;
+use solana_sdk::account::Account;
+use solana_sdk::program_option::COption;
+use spl_token_2022::state::Mint;
+use spl_token_wrap::state::Backpointer;
+use spl_token_wrap::{
+    get_wrapped_mint_address, get_wrapped_mint_authority, get_wrapped_mint_backpointer_address,
+};
+use {
+    mollusk_svm::{result::Check, Mollusk},
+    solana_program::pubkey::Pubkey,
+    spl_token_wrap::instruction::create_mint,
+};
+
+const MINT_DECIMALS: u8 = 12;
+const MINT_SUPPLY: u64 = 500_000_000;
+const FREEZE_AUTHORITY: &str = "11111115q4EpJaTXAZWpCg3J2zppWGSZ46KXozzo9";
```

Comment:
nit: may as well initialize this as a const Pubkey rather than using `from_str_const` for every test